go/enclave/storage/init/edgelessdb/002_unique_ky.sql

```diff
@@ -0,0 +1,6 @@
+SET SESSION  rocksdb_max_row_locks = 10000000;
+DELETE s1 FROM statedb32 s1 INNER JOIN statedb32 s2 ON s1.ky = s2.ky AND s1.id > s2.id;
+ALTER TABLE statedb32 ADD CONSTRAINT unique_ky_statedb32 UNIQUE (ky);
+
+DELETE FROM statedb64  WHERE id NOT IN (SELECT MIN(id) FROM statedb64 GROUP BY ky);
```

Comment:
The AI told me this will fail in mysql with an error like this:

“You can’t specify target table ‘statedb64’ for update in FROM clause”
because the subquery reads from the same table you’re deleting from:

Apparently have to write it out like your other one with a join:
```
DELETE s1
FROM statedb64 s1
JOIN statedb64 s2
  ON s1.ky = s2.ky
 AND s1.id > s2.id;
```